databroker/broker.py

```diff
@@ -575,6 +545,68 @@ def get_events(self, header, pvs):
                 yield Document('Event', doc)
 
 
+class DangerBroker(Broker):
```

Comment:
Any reason to have a new inherited class? Can we just put this function inside Broker class?